input/resources/au-servicerequest.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="au-servicerequest" />
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="0"/> 
+  </extension>
+  <url value="http://hl7.org.au/fhir/StructureDefinition/au-servicerequest" />
+  <name value="AUBaseServiceRequest" />
+  <title value="AU Base Service Request" />
+  <status value="draft" />
+  <description value="This profile defines a service request structure that localises core concepts for use in an Australian context." />
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="ServiceRequest" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/ServiceRequest" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="ServiceRequest">
+      <path value="ServiceRequest" />
+      <short value="A service request in an Australian healthcare context" />
+    <definition value="A record of a request for service such as diagnostic investigations, treatments, or operations to be performed." />
+    </element>
+    <element id="ServiceRequest.extension">
+      <path value="ServiceRequest.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="ServiceRequest.extension:targetBodyStructure">
+      <path value="ServiceRequest.extension" />
+      <sliceName value="targetBodyStructure" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/procedure-targetBodyStructure" />
+      </type>
+    </element>
+    <element id="ServiceRequest.identifier">
+      <path value="ServiceRequest.identifier" />
+      <short value="Identifiers assigned to this order" />
+      <definition value="Identifiers assigned to this order instance by the orderer and/or the receiver and/or order fulfiller." />
+      <type>
+        <code value="Identifier" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/Identifier" />
+        <profile value="http://hl7.org.au/fhir/StructureDefinition/au-localorderidentifier" />
+      </type>
+    </element>
+    <element id="ServiceRequest.code">
+      <path value="ServiceRequest.code" />
+      <binding>
+        <strength value="example" />
+        <valueSet value="https://healthterminologies.gov.au/fhir/ValueSet/procedure-1" />
+      </binding>
+    </element>
+    <element id="ServiceRequest.performerType">
+      <path value="ServiceRequest.performerType" />
+      <binding>
```

Comment:
This is inherited from core ServiceRequest resource, no need to include in the profile.

Reply:
Removed, thanks.

Reply:
Confirmed